### ISSUE TITLE
bug: missing infos for the `useRunDetails` hook

### ISSUE THREAD

**Issue by wladpaiva** [triggerdotdev/trigger.dev#870]
### Provide environment information

  System:
    OS: macOS 14.2.1
    CPU: (10) arm64 Apple M2 Pro
    Memory: 227.30 MB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 20.10.0 - ~/.nvm/versions/node/v20.10.0/bin/node
    Yarn: 1.22.21 - ~/.nvm/versions/node/v20.10.0/bin/yarn
    npm: 10.2.3 - ~/.nvm/versions/node/v20.10.0/bin/npm
    bun: 1.0.25 - ~/.bun/bin/bun

### Describe the bug

- `data.subtasks` are in the typescript but the hook doesn't spit them out
- `status` are not showing WAITING... haven't tested the others

### Reproduction repo

https://discord.com/channels/1066956501299777596/1066956844553207828/1200203785457377390

### To reproduce

basically
```tsx
  const {data, error} = useRunDetails(runId)
  return  <pre>{JSON.stringify(data, null, 2)}</pre>
```

and this job
https://trigger.dev/docs/documentation/concepts/tasks#extracting-common-tasks

**Comment by matt-aitken** [comment]
@wladpaiva to get subtasks in the result you need to pass `subtasks: true` in the options.

```ts 
const { isLoading, isError, data, error } = useEventRunDetails(id, { subtasks: true });
```
